src/discussions/learners/LearnersView.jsx

```diff
@@ -60,8 +60,7 @@ const LearnersView = () => {
       courseConfigLoadingStatus === RequestStatus.SUCCESSFUL && learners.map((learner) => (
         <LearnerCard learner={learner} key={learner.username} />
       ))
-    // eslint-disable-next-line react/jsx-no-useless-fragment
-    ) || <></>
+    ) || null
```

Comment:
update this function 
const renderLearnersList = useMemo(() => {
  if (courseConfigLoadingStatus === RequestStatus.SUCCESSFUL) {
    return learners.map((learner) => <LearnerCard learner={learner} key={learner.username} />);
  }
  return null;
}, [courseConfigLoadingStatus, learners]);